components/tikv_util/src/metrics/threads_linux.rs

```diff
@@ -287,6 +289,116 @@ lazy_static! {
     };
 }
 
+#[inline]
+fn get_name(command: &str) -> String {
+    if command != "" {
+        return command.to_owned();
+    }
+    String::from("anony")
+}
+
+/// Use to collect cpu usages and disk I/O rates
+pub struct ThreadInfoStatistics {
```

Comment:
can it work on Mac OSX, or only Linux?

Reply:
Only for Linux now.

Reply:
so you need to use `#[cfg(target_os = "linux")]` explicitly.

Reply:
OK, I'll fix it.